courses/tests.py

```diff
@@ -323,3 +323,41 @@ def test_course_no_good_sections(self):
     def test_not_get_course(self):
         response = self.client.get('/all/courses/CIS-160/')
         self.assertEqual(response.status_code, 404)
+
+
+@override_settings(SWITCHBOARD_TEST_APP='api')
+class RelocateReqsRestsTest(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+        class Rest:
+            requirement_description = ''
+        self.Rest = Rest
+        self.rests = [Rest()]
+        self.reqs = []
+
+    def test_bfs(self):
+        self.rests[0].requirement_description = 'Benjamin Franklin Seminars'
+        relocate_reqs_from_rests(self.rests, self.reqs)
+        self.assertEqual(self.reqs, ['Benjamin Franklin Seminars'])
+
+    def test_nsm(self):
+        self.rests[0].requirement_description = 'Natural Science & Math Sector'
+        relocate_reqs_from_rests(self.rests, self.reqs)
+        self.assertEqual(self.reqs, ['Natural Science & Math Sector'])
+
+    def test_hss(self):
+        self.rests[0].requirement_description = 'Humanities & Social Science Sector'
+        relocate_reqs_from_rests(self.rests, self.reqs)
+        self.assertEqual(self.reqs, ['Humanities & Social Science Sector'])
+
+    def test_mixed(self):
```

Comment:
Can you add a test where there's a restriction that's not one of these three? Just to make sure that it doesn't add them in by accident.

Also a test where there's already something in `self.reqs` list in addition to the ones being added. Just to cover other edge cases.